templates/cjs/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -893,10 +894,23 @@ export class {{ service.name }}Client {
         {{ util.toInterface(method.outputInterface) }}
       ]>|void {
     request = request || {};
+  {%- if method.maxResultsParameter %}
+    // the underlying API expects a UInt32Value or Int32Value object
```

Comment:
I would actually get rid of this comment or perhaps reword it, as technically this is what goes in the client library (and thus what users actually use). Maybe just say:

```
Converts number to Unit32 or Int32 value for non-compliant APIs.
```

Reply:
Done